### DISCUSSION TITLE
Reset main branch to tag 3.0.0

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#12536] [General]
@bokeh/core I'd like to propose resetting the `main` branch to point at the 3.0.0 tag:
```
git checkout main

git reset --hard 3.0.0
```
There are various project files like the changelog, security policy, etc. that people might try to stumble and access from `main`. We had in fact in the past planned to keep `main` pointing at the latest release tag, just in a more gentle fashion. This way is blunt, and will require a force-push to main, which I now think is acceptable. I.e., I do think having `main` point at the latest release is a good thing in general [1], but since `main` is not used for any development, it's just not worth the effort to do things "nicely". 

[1] we could potentially even automate this as part of release-deploy

**Comment by mattpap** [comment]
I'm pretty sure that we already agreed on something like this before. In any case, definitely let's do this.

**Reply by bryevdv** [reply]
I think we agreed to try to keep things updated more gently, via merging patches, but at this point things are so far diverged that it does not seem worthwhile to bother. In any case I wanted to raise the discussion for wider visibility first, before taking unilateral action.  

Assuming there are no concerns or objections raised in the next day, I'll execute the reset later this week.

**Reply by ianthomas23** [reply]
I have no objections.

**Comment by bryevdv** [comment]
This is done.

**Reply by bryevdv** [reply]
And also now for 3.0.1
